index.d.ts

```diff
@@ -25,7 +25,7 @@ namespace Speaker {
  *
  * @param opts options.
  */
-class Speaker extends Writable {
+export default class Speaker extends Writable {
```

Comment:
This doesn't accurately reflect how the JavaScript code is exported, could you instead only change `namespace Speaker` to `declare namespace Speaker`, and leave the export as `export = Speaker`?

Reply:
Unfortunately the export is not going to work like that, the `export =` syntax causes TS to error with:

> Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier. ts(1046)

Reply:
You could just do `declare class Speaker extends Writable {` and then leave the `export = Speaker`. That seems to keep things working as expected, while fixing the errors.